lib/maropost/api.rb

```diff
@@ -0,0 +1,95 @@
+module Maropost
+  class Api
+    def self.find(email)
+      url = "#{Maropost.configuration.api_url}/contacts/email.json?contact[email]=#{email}"
+
+      response = RestClient::Request.execute(
```

Comment:
hey Serene...we've wrapped `RestClient` requests so that they do some additional logging (https://rubygems.org/gems/rest-client-jogger)...could you update these `RestClient::Request.execute`s to use that? Here's an example (but @jessedoyle might be able to let you know how we are doing it in the new DE app): 

https://github.com/amaabca/ama_oauth_client/blob/master/lib/ama_oauth_client/sudo_mode/model.rb#L5

Then to get the logging you just have to add this to the app (if it isn't there already):

`RestClient::Jogger::EventSubscriber.new.subscribe`